src/DashboardUI/src/pages/application/review/map/ReviewMap.tsx

```diff
@@ -59,9 +67,12 @@ function ReviewMap(props: ReviewMapProps) {
       return;
     }
 
-    // display user-drawn polygons on map
+    // display user-drawn polygons + control location point on map
     // side note: labels are handled separately
-    setUserDrawnPolygonData(userDrawnPolygonFeatures);
+    const allFeatures: Feature<Geometry, GeoJsonProperties>[] = (
+      userDrawnPolygonFeatures as Feature<Geometry, GeoJsonProperties>[]
+    ).concat(controlLocationFeature ? [controlLocationFeature] : []);
```

Comment:
Little bit hard to read imo. Thoughts on this?
```suggestion
    const controlLocationFeatures = controlLocationFeature ? [controlLocationFeature] : [];
    const allFeatures: Feature<Geometry, GeoJsonProperties>[] = [
      ...userDrawnPolygonFeatures,
      ...controlLocationFeatures,
    ];
```

Reply:
works great, updating